client/components/fields/ColoredValueSelectField/style.scss

```diff
@@ -0,0 +1,19 @@
+@import '~superdesk-core/styles/sass/mixins.scss';
+@import '~superdesk-core/styles/sass/variables.scss';
+
+.ColoredValueSelect__popup {
+	position: relative;
+    z-index: 2000;
+    background: #fff;
+    padding: 10px 10px 10px 10px;
+    @include box-shadow(0 0 5px 0 rgba(0,0,0,0.45));
+    @include border-radius(3px);
+    max-width: 200px;
+    @include border-box();
+
+    button {
+    	padding: .6rem 1.6rem;
+    	width: 100px;
+    	text-align: left;
+    }
+}
```

Comment:
It would be good to have the `<li>` options highlight on mouse over like in the rest of Superdesk